src/components/Layout/Topbar/TopBar.jsx

```diff
@@ -0,0 +1,53 @@
+import { HStack, Spacer, Icon, Heading, Flex } from '@chakra-ui/react';
+import PropTypes from 'prop-types'; // ES6
+import { FaTeamspeak } from 'react-icons/fa';
+
+import TopBarLink from './TopBarLink';
+
+import { useAuthContext } from 'context/AuthContext.jsx';
+
+function TopBar({ setIsLogged }) {
```

Comment:
I'd suggest moving `setIsLogged` to `AuthContext` too

```suggestion
function TopBar() {
```